### PR TITLE
feat(scripts): bulk update/remove internal API endpoints

### PR DESCRIPTION
**Pull request by saraghds**
Closes https://github.com/influxdata/idpe/issues/15643

This PR adds the new API endpoints for the scripts service. These APIs are not exposed on the edge and is only called internally by other services. This means they don't belong inside the existing `svc-invocable-scripts.yml` file, so I've created a new folder under `src/svc` called `invocable-scripts-internal` to include the paths and the schemas. And this folder is not added to the `generate.sh` script so that it won't be published.

### CODE REVIEW

**Review by glinton — APPROVED** [review]
I like how you defined only the unique internal paths, as opposed to redefining the entire scripts api.

File: src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+patch:
+  operationId: PatchScriptsBulkRemoveLabel
+  tags:
+    - Data I/O endpoints
+    - Invokable Scripts
+  summary: Removes a label from all scripts in the organization.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml:6]:
```suggestion
  summary: Remove a label from all scripts in the organization.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  description: |
+    Removes a label from all scripts in the organization.
+    This is a global operation and can only be called
+    internally between services. Global (not org-scoped)
+    access needed for `labels` and `functions`.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml:11]:
```suggestion

   #### Limitations
   
    - This is a global operation that can only be called
    internally between services.
    
    #### Permissions
    
    - Requires global (not org-scoped) permissions for `labels` and `functions`.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../schemas/ScriptBulkRemoveLabelRequest.yml'
+  responses:
+    '204':
+      description: |
+        Success.
+        The label is removed from all scripts of the organization.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml:23]:
```suggestion
        The label is removed from all scripts owned by the organization.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+patch:
+  operationId: PatchScriptsBulkUpdateLabel
+  tags:
+    - Data I/O endpoints
+    - Invokable Scripts
+  summary: Updates a label name in all scripts in the organization.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml:6]:
```suggestion
  summary: Update a label name for all scripts in the organization.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  description: |
+    Updates a label name in all scripts in the organization.
+    This is a global operation and can only be called
+    internally between services. Global (not org-scoped)
+    access needed for `labels` and `functions`.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml:11]:
```suggestion
    Updates a label name for all scripts in the organization.

   #### Limitations
   
    - This is a global operation that can only be called
    internally between services.
    
    #### Permissions
    
    - Requires global (not org-scoped) permissions for `labels` and `functions`.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../schemas/ScriptBulkUpdateLabelRequest.yml'
+  responses:
+    '204':
+      description: |
+        Success.
+        The label name is updated in all scripts of the organization.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml:23]:
```suggestion
        The label name is updated for all scripts owned by the organization.
```

File: src/svc/invocable-scripts-internal/schemas/ScriptBulkRemoveLabelRequest.yml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+type: object
+properties:
+  orgID:
+    description: The organization ID of the scripts for removing the label.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/schemas/ScriptBulkRemoveLabelRequest.yml:4]:
```suggestion
    description: The ID of the organization that owns the scripts with the label.
```

File: src/svc/invocable-scripts-internal/schemas/ScriptBulkUpdateLabelRequest.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+type: object
+properties:
+  orgID:
+    description: The organization ID of the scripts for updating the label name.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/schemas/ScriptBulkUpdateLabelRequest.yml:4]:
```suggestion
    description: The ID of the organization that owns the scripts with the label.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    #### Limitations
+
+    - This is a global operation that can only be called
+    internally between services.
+
+    #### Permissions
+
+    - Requires global (not org-scoped) permissions for `labels` and `functions`.
+  requestBody:
+    description: The org ID and the label names to remove.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_remove.yml:19]:
```suggestion
    description: |
       The label to remove from all scripts in the organization.
```

File: src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    #### Limitations
+
+    - This is a global operation that can only be called
+    internally between services.
+
+    #### Permissions
+
+    - Requires global (not org-scoped) permissions for `labels` and `functions`.
+  requestBody:
+    description: The org ID and the label name to update.
```
Comment (jstirnaman) [inline src/svc/invocable-scripts-internal/paths/scripts_labels_update.yml:19]:
```suggestion
    description: The label update to apply to all scripts in the organization.
```

**Review by jstirnaman — APPROVED** [review]
Some minor suggestions. Don't let me hold you up; we can come back to it. Thanks for asking!

**Pull request merged**